### PR TITLE
change: remove comments from program.cs

### PR DESCRIPTION
**Pull request by TihomirAleksandrov**
Removed unnecessary comments from program.cs

### CODE REVIEW

File: BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs [hunk 1]
```diff
@@ -6,13 +6,12 @@
 
 namespace BulgarskiChiflikGuesthouse
 {
-    public class Program
+	public class Program
```
Comment (AlbertEdward) [inline BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs:9]: Question: why has this been changed?

File: BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs [hunk 2]
```diff
@@ -6,13 +6,12 @@
...
     {
         public static void Main(string[] args)
         {
             var builder = WebApplication.CreateBuilder(args);
 
-            // Add services to the container.
```
Comment (IvHrIvanov) [inline BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs:15]: Good move

File: BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs [hunk 3]
```diff
@@ -6,13 +6,12 @@
 
 namespace BulgarskiChiflikGuesthouse
 {
-    public class Program
```
Comment (AlbertEdward) [inline BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs:9]: опса

File: BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs (hunk 1 above)
Comment (TihomirAleksandrov) [reply BulgarskiChiflikGuesthouse/BulgarskiChiflikGuesthouse/Program.cs:9]: The code is auto adjusted.

**Pull request merged**